### ISSUE TITLE
unable to open image: File name too long

### ISSUE THREAD

**Issue by mviereck** [ImageMagick/ImageMagick#5816]
### ImageMagick version

7.1.0-51 Q16-HDRI x86_64 1cf41df5d:20221013

### Operating system

Linux

### Operating system, version and so on

Debian bullseye

### Description

I am creating images with sometimes very long file names.
It seems I've hit a limit today:
```
magick: unable to open image './cache.imfuse/substackresult.--col=C=Jzazbz,c0--diffstat=w18.75,r4,Median,Mean--statistic=w18.75,r2,StandardDeviation--log=w18.75,s1--wavelet=w18.75,p5--cmd=w25,C=HSL,c2,neg,level=no--maskwave=p20--maskblur=s1--level=all--cutless=p25--cutwave=p100--cutmorph=r1,R1,Erode_md740ec4.substack1.mpc': File name too long @ error/blob.c/OpenBlob/3570.
```
Short:
```
magick: unable to open image [...] File name too long @ error/blob.c/OpenBlob/3570.
```
Is there a way to bypass this limit? Except using shorter names, obviously?
I assume the same error would occur with long pathes.

Basically I consider this to be a bug, because at least long pathes are not unusual.
I am not aware of a general limit of file name length; at least a general limit will be much longer than the given examples.

### Steps to Reproduce

Try to write an image with a very long file name:
```
magick rose: "$(seq -s '' 200).jpg"
```

**Comment by dlemstra** [comment]
You are hitting an OS limit, that error message is from your operating system and not from us. You are probably hitting a 255 bytes limit (https://en.wikipedia.org/wiki/Comparison_of_file_systems#Limits).

**Comment by mviereck** [comment]
Thank you!
Indeed, I did not even assume that this could be a file system limit. I should have checked this before asking here, sorry.

**Comment by dlemstra** [comment]
I also did not know there was a file name length limit enforced by the OS, so today I learned. Thank you 😁
